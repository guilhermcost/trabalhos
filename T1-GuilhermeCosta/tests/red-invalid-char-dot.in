// Testa caracter invalido .
void main(void) {
    1 r.;
}
