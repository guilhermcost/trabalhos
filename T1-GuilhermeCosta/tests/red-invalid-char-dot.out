(2,KEY,"void")
(2,ID,"main")
(2,SYM,"(")
(2,KEY,"void")
(2,SYM,")")
(2,SYM,"{")
(3,NUM,"1")
(3,ID,"r")
(3,ERROR,".")
(3,SYM,";")
(4,SYM,"}")
